### ISSUE TITLE
Compare services-flake with `numtide/devshell`

### ISSUE THREAD

**Issue by shivaraj-bh** [juspay/services-flake#151]
https://discourse.nixos.org/t/announcing-services-flake/41338/4?u=shivaraj-bh

**Comment by srid** [comment]
> think another contender would be [numtide/devshell 6](https://github.com/numtide/devshell), which is more open to use as a component (unlike devenv), but **doesn’t provide ready-to-use services**.

Looks like there's nothing to compare against?

**Comment by shivaraj-bh** [comment]
Right, they only seem to have postgres service: https://github.com/numtide/devshell/tree/main/extra/services

**Comment by srid** [comment]
Oh so they have *something*. We can add it to the table then.

**Comment by oneingan** [comment]
I think the point here will be that devshell could use services definitions from here, but I suspect that this project is tightly coupled to process-compose-flake

**Comment by srid** [comment]
Another candidate: `Devbox`

https://www.jetify.com/devbox/docs/guides/services/

via https://x.com/meijer_s/status/1791588665669337485
